docker-images/operator/scripts/launch_java.sh

```diff
@@ -30,6 +30,21 @@ calc_max_memory() {
   calc_maximum_size_opt "${mem_limit}" "20"
 }
 
+check_cgroups_v2() {
+  local mem_file="$1"
+  local num_pattern='^[0-9]+([.][0-9]+)?$'
+
+  local calc_cgroups_mem=false
+  if [[ -r "${mem_file}" ]]; then
+    value=$(cat "${mem_file}")
+    if [[ $value =~ $num_pattern ]]; then
+      calc_cgroups_mem=true
+    fi
+  fi
+
+  echo $calc_cgroups_mem
+}
```

Comment:
This check seems wrong based on the docs, I would expect you to check specifically for `max`.

Reply:
Well, I thought that there could be also different values than just `max` - that's why I'm checking, if the file contains a number.

Reply:
Okay, I will check what the `memory.high` contains when I will not specify any resources limits inside the deployment and let you know my findings